api/node/index.ts

```diff
@@ -171,6 +173,20 @@ export abstract class Model<T> {
         this.notify = new NullPeer();
     }
 
+    /**
+     * Returns a new Model where all elements are mapped by the function `mapFunction`.
+     * @template T the type of the source model's items.
+     * @param mapFunction functions that maps
```

Comment:
Shouldn't this be singular?

And perhaps mention that it makes from T to U.

Reply:
done